doc/specs/QuickLauncher.md

```diff
@@ -0,0 +1,398 @@
+that search is not fast enough and it does not give results that are
+relevant to them. This issue received the fourth most thumbs up (with
+70+ thumbs up) in the category of suggested PowerToy indicating that
+users are interested in this PowerToy.
+
+### 1.3.1 Survey Results
+
+PowerToys team sent a survey to Windows Developer community to gain preliminary insights on users’ current launch experience. Here are some key takeaways:
+
+1.  Nearly 70% of respondents want “run as administrator” to be a
+    feature in an application launcher. The next most requested feature
+    is auto-complete text suggestions.
+
+2.  Less than 15% of respondents who use Windows as their main OS use
+    third-party tools as a part of their launch experience.
+
+3.  Nearly 65% of respondents enters an application name when searching.
+
+4.  All respondents who use MacOS as their main OS and use third-party tools to launch an
+    application runs Alfred.
+
+5.  For respondents who use Search on Windows, performance and accuracy
+    of search result are two top most requested improvements that
+    they would like to see.
+
+## 1.4. Existing Solutions or Expectations
+
+Currently, power users can launch an application using Windows launcher by
+running Win+R shortcut or searching the desired application in Windows
+Search (Win+S, Windows menu + start typing). Power users can also search
+for results through Windows Search by typing in the search bar directly
+or by specifying a category (apps, documents, email, web, folders,
+music, people, photos, settings, and videos). Several third-party tools
+also exist to extend the capability of Windows’ launcher.
+
+Here is a matrix which compares features between third-party tools:
+
+| Tools/Features     | Text suggestion    | App suggestion     | Open in terminal  | Run as admin       | Saved history from previous session | Open file location |
+|--------------------|--------------------|--------------------|-------------------|--------------------|-------------------------------------|--------------------|
+|[Alfred (free)](https://www.alfredapp.com/)       | ![alt text][cross] | ![alt text][check] |![alt text][cross] | ![alt text][cross] | ![alt text][check]                  | ![alt text][cross] |
+|[Spotlight](https://support.apple.com/en-us/HT204014)</p>       | ![alt text][check] | ![alt text][check] |![alt text][cross] | ![alt text][cross] | ![alt text][check]                  | ![alt text][check] |
+|[Listary](https://www.listary.com/)</p>         | ![alt text][cross] | ![alt text][check] |![alt text][cross] | ![alt text][check] shown as a separate cmd | ![alt text][cross]                  | ![alt text][check] on right click |
+|[Wox](http://www.wox.one/)</p>             | ![alt text][cross] | ![alt text][check] |![alt text][check] via “>” or Win+R | ![alt text][check] on right click | ![alt text][check]                  | ![alt text][check] on right click|

```

Comment:
Can I point out that on Wox you can access the "run as administrator" and "open file location" with Shift+Enter, not just right click? The whole point of a launcher like this is to be able to keep both hands on the keyboard

Reply:
Thanks for pointing this out! I've added it to the comparison table.